headings for financial analysis (#19)
vincent-l-j/project-1: README.md

pip install -r requirements.txt
streamlit run app.py
```
## Summary of Findings

This should include a heading for each question that was asked of your data, with a short description of what you found and any relevant plots under each heading.

> 

Reviewer: ```suggestion

The following analysis was performed using a chosen portfolio of [`BTC`, `ETH`, `SOL`] with weightings of [60, 20, 20] respectively.
```

> ### How has your portoflio performed in the past year?

Reviewer: ```suggestion
### How has your portfolio performed in the past year?
```

### How has your portfolio performed cumulatively in the past year?

> ### How storngly correlated is your portfolio to BTC?

Reviewer: ```suggestion
### How strongly correlated is your portfolio to BTC?
```

> ### How volitile is your portfolio?

Reviewer: ```suggestion
### How volatile is your portfolio?
```

> ### How will you portoflio perform in one year?

Reviewer: ```suggestion
### How will your portfolio perform in one year?
```

> ### How will your portfolio perform in one year?

Reviewer: ```suggestion
### How might your portfolio perform in one year?
```

> ### What is the most likely range of your returns?

Reviewer: ```suggestion
### What is the most probable expected portfolio return in one year?
```

> ### Is the portfolio performing adequetely?

Reviewer: ```suggestion
### Is the portfolio performing adequately?
```

> ### What weighting is optimal for minimum volitility

Reviewer: ```suggestion
### What weighting is optimal for minimum volatility?
```

> ### What weighting is optimal for minimum volatility?

Reviewer: ```suggestion
### Find the portfolio with the minimum volatility
```

> ### What weighting is optimal for maximum sharpe ratio?

Reviewer: ```suggestion
### Find the portfolio with the maximum Sharpe Ratio, also called the tangency portfolio
```

> ### How to find the portfolio with the maximum Sharpe Ratio for a given target Volatility

Reviewer: ```suggestion
### Given a target volatility of 0.85, find the portfolio with the maximum Sharpe Ratio
```

> ### How to find the portfolio with the minimum volatility for a given target return?

Reviewer: ```suggestion
### Given an expected return of 1.5, find the portfolio with the minimum volatility
```

pip install -r requirements.txt
streamlit run app.py
```
## Summary of Findings

This should include a heading for each question that was asked of your data, with a short description of what you found and any relevant plots under each heading.

> The following analysis was performed using a chosen portfolio of [`BTC`, `ETH`, `SOL`] with weightings of [60, 20, 20] respectively.

Reviewer: ```suggestion
The following analysis was performed using a chosen portfolio of [`BTC`, `ETH`, `SOL`] with weightings of [60, 20, 20] respectively.

```

Reviewer: whitespace

pip install -r requirements.txt
streamlit run app.py
```

> ## Summary of Findings

Reviewer: ```suggestion

---

## Summary of Findings
```

pip install -r requirements.txt
streamlit run app.py
```

---

## Summary of Findings

This should include a heading for each question that was asked of your data, with a short description of what you found and any relevant plots under each heading.

The following analysis was performed using a chosen portfolio of [`BTC`, `ETH`, `SOL`] with weightings of [60, 20, 20] respectively.

### How has your portfolio performed in the past year?

### How has your portfolio performed cumulatively in the past year?

### How strongly correlated is your portfolio to BTC?

### How volatile is your portfolio?

### How might your portfolio perform in one year?

### What is the most probable expected portfolio return in one year?

### Is the portfolio performing adequately?

### Find the portfolio with the minimum volatility

### Find the portfolio with the maximum Sharpe Ratio, also called the tangency portfolio

### Given a target volatility of 0.85, find the portfolio with the maximum Sharpe Ratio

### Given an expected return of 1.5, find the portfolio with the minimum volatility

> 

Reviewer: ```suggestion
```
